packages/integration-sdk-entities/README.md

```diff
@@ -0,0 +1,32 @@
+# @jupiterone/integration-sdk-entities
+
+This package contains all entities generated from J1's data model to be used
+throughout J1 systems.
```

Comment:
I'm okay with it being entities specific, but the `data-model` technically defines the schema for all graph objects including relationships.

Reply:
Made it more explicit